src/DIRAC/Core/DISET/private/Service.py

```diff
@@ -360,6 +360,16 @@ def handleConnection(self, clientTransport):
       self._monitor.setComponentExtraParam('queries', self._stats['connections'])
     # TODO: remove later
     if useThreadPoolExecutor:
+      nQueued = self._threadPool._work_queue.qsize()
```

Comment:
I know this is using a private member variable but it's already used elsewhere in this file.

Reply:
I am amazed that pylint does not complain...

Reply:
We run with `-E` in the CI